package.json

```diff
@@ -8,14 +8,14 @@
   "scripts": {
     "start": "craco start",
     "build": "yarn i18n:compile && craco build && yarn writeVersion",
-    "test": "NODE_PATH=src/custom craco test --env=jsdom",
+    "test": "craco test --env=jsdom",
     "start:ssl": "HTTPS=true yarn start",
     "start:service-worker": "yarn build && yarn serve",
     "mock": "REACT_APP_MOCK=true yarn start",
     "build:analyze": "cross-env REACT_APP_ANALYZE_BUNDLE=true yarn build",
     "ipfs:build": "cross-env PUBLIC_URL=\".\" yarn build",
     "ipfs:publish": "ipfs-deploy build -p pinata -O",
-    "test:debug": "NODE_PATH=src/custom craco --inspect-brk test --runInBand --no-cache",
```

Comment:
no need to change, but we might not need craco soon, it was introduced just to do the overlay layer on top of Uni